src/angularjs/src/app/components/navbar/navbar.directive.js

```diff
@@ -35,7 +31,11 @@
             templateUrl: 'app/components/navbar/navbar.html',
             controller: 'NavbarController',
             controllerAs: 'navbar',
-            bindToController: true
+            bindToController: true,
+            scope: {
+                admin: '@',
```

Comment:
Fine to start, but this is state you could theoretically acquire from an AuthService of some kind.